Server/testPlanExternalAuth.md

```diff
@@ -0,0 +1,49 @@
+| Having login credentials, save in session enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Mount a system wide SFTP folder using RSA public key credentials, generate it if neccesary | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Mount a system wide Amazon_S3 folder using login credentials, save in session | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Mount a system wide Amazon_S3 folder using login credentials, save in DB | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Having login credentials, save in DB enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Having login credentials, save in session enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Mount a system wide Amazon_S3 folder using RSA public key credentials, generate it if neccesary | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Mount a system wide Dropbox folder using login credentials, save in session | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Mount a system wide Dropbox folder using login credentials, save in DB | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Having login credentials, save in DB enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Having login credentials, save in session enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Mount a system wide Dropbox folder using RSA public key credentials, generate it if neccesary | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Mount a system wide Google_Drive folder using login credentials, save in session | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Mount a system wide Google_Drive folder using login credentials, save in DB | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
+| Having login credentials, save in DB enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Having login credentials, save in session enabled, login with a user which has no permission to connect| User cannot see mounted folder. | :construction: |
+| Mount a system wide Google_Drive folder using RSA public key credentials, generate it if neccesary | Folder is mounted, green dot appears, files can be uploaded to mounted folder. | :construction: |
```

Comment:
also no RSA key credentials for GDrive.
You might want to check the auth options for the different types in the UI.